### DISCUSSION TITLE
How to compile for win 32bits - I'm using windows 10 64bit

### DISCUSSION THREAD

**Discussion by 0-10V** [vlang/v#17594] [Questions and Answers]
I need to use a 32 bit DLL so I need to compile an 32bit executable.
How to do this?

I also tried to compile V itself to be 32bit with no success.

Note: No I can not compile the DLL in 64bit or request a 64bit version of the DLL. It's a  DLL used on automotive application (CCP - CAN calibration protocol) whose standard itself demands a 32bit DLL.

**Comment by JalonSolov** [comment]
You can _try_ adding `-m32` along with `-shared` to the command line when you compile.

I don't know if it will work... I don't know if anyone on the V team _has_ a 32 bit system to test this on - I know nobody uses 32 bit regularly.
